orbit/pkg/scripts/exec_nonwindows.go

```diff
@@ -28,13 +28,17 @@ func execCmd(ctx context.Context, scriptPath string) (output []byte, exitCode in
 	cmd := exec.CommandContext(ctx, "/bin/sh", scriptPath)
 
 	if directExecute {
-		err = os.Chmod(scriptPath, 0766)
+		err = os.Chmod(scriptPath, 0700)
```

Comment:
To double-check: this will also affect scripts, it's ok to have them non-world-readable too? (assuming yes but I don't remember why we made them world-readable to begin with)

Reply:
When I originally wrote the direct execute portion I chose file permissions that I thought made sense, but I had no solid reasoning for why I chose it. I realized after that only one user should really need to be able to execute and read the scripts (fleetd/root). I can put it back if you would like

Reply:
No that sounds good to me! Just wanted to make sure there wasn't a strong constraint or something.